spec/components/intervention_spec.rb

```diff
@@ -19,6 +19,42 @@ def component_name
     assert_select ".gem-c-intervention .govuk-link", text: /Hide this suggestion/
   end
 
+  it "renders the component with GA4 tracking when ga4_tracking is true" do
+    render_component(
+      name: "test-campaign",
+      suggestion_text: "You might be interested in",
+      suggestion_link_text: "Travel abroad",
+      suggestion_link_url: "/travel-abroad",
+      dismiss_text: "Hide this suggestion",
+      ga4_tracking: true,
+    )
+
+    assert_select ".gem-c-intervention[data-ga4-intervention-banner]"
+    assert_select ".gem-c-intervention a:first-of-type[data-module='ga4-link-tracker']"
```

Comment:
I think there was some code to include this module with an already existing module, should there be a test for that scenario?

Reply:
Good idea, done :+1:

Reply:
Nice.